data/class/helper/SC_Helper_Purchase.php

```diff
@@ -260,7 +260,8 @@ public static function getOrderTemp($uniqId)
     {
         $objQuery = SC_Query_Ex::getSingletonInstance();
 
-        return $objQuery->getRow('*', 'dtb_order_temp', 'order_temp_id = ?', [$uniqId]);
+        $result = $objQuery->getRow('*', 'dtb_order_temp', 'order_temp_id = ?', array($uniqId));
```

Comment:
```suggestion
        $result = $objQuery->getRow('*', 'dtb_order_temp', 'order_temp_id = ?', [$uniqId]);
```
#996 より、short syntax に統一しているため修正をお願いします

Reply:
@nanasess 
short syntaxの件、testの件承知しました。確かに最近short syntaxで書かれているな、と思っていました。

ところでですが、このような些細な修正（？）の時にもissueをたてたほうがいいか、悩ましいところです。。。

Reply:
@clicktx ありがとうございます！ PR のみで済むものであれば、 issue なしでも問題ございません。

Reply:
@nanasess ありがとうございます。issueなしでもokとの事、承知しました！